cmd/exec.go

```diff
@@ -50,67 +51,35 @@ func execRun(cmd *cobra.Command, args []string) error {
 		})
 	}
 
-	env := environ(os.Environ())
+	env := environ.Environ(os.Environ())
 	secretStore, err := getSecretStore()
 	if err != nil {
 		return errors.Wrap(err, "Failed to get secret store")
 	}
-	envVarKeys := make([]string, 0)
 	for _, service := range services {
 		if err := validateService(service); err != nil {
 			return errors.Wrap(err, "Failed to validate service")
 		}
 
-		rawSecrets, err := secretStore.ListRaw(strings.ToLower(service))
+		collisions := make([]string, 0)
+		var err error
+		if _, noPaths := os.LookupEnv("CHAMBER_NO_PATHS"); noPaths {
+			err = env.LoadNoPaths(secretStore, service, &collisions)
+		} else {
+			err = env.Load(secretStore, service, &collisions)
+		}
 		if err != nil {
 			return errors.Wrap(err, "Failed to list store contents")
```

Comment:
Can we just return `err` here? Since we already return `errors.Wrap(err, "Failed to list store contents")` from `Load` and `LoadNoPaths`

Reply:
Yep, good point. Updated to return err directly from `Load*`